Plan/sponge/src/main/java/com/djrapitops/plan/storage/file/SpongePlanFiles.java

```diff
@@ -50,9 +54,25 @@ public SpongePlanFiles(
     @Override
     public Resource getResourceFromJar(String resourceName) {
         try {
-            return new SpongeAssetResource(resourceName, Sponge.getAssetManager().getAsset(plugin, resourceName).orElse(null));
-        } catch (IllegalStateException spongeNotEnabled) {
+            if (!(plugin instanceof PlanSponge)) {
```

Comment:
Is this check for testing? 🤔

Reply:
It's been a while since I wrote this but, tests do indeed fail if that check isn't there, but they pass if it is.

So I'm not sure if we should be worried that it passes when there's presumably a exception being thrown.

Test fail if it's casted without a instanceof check,
```
java.lang.ClassCastException: class com.djrapitops.plan.PlanPlugin$MockitoMock$xLnK0G63 cannot be cast to class com.djrapitops.plan.PlanSponge (com.djrapitops.plan.PlanPlugin$MockitoMock$xLnK0G63 and com.djrapitops.plan.PlanSponge are in unnamed module of loader 'app')
	at com.djrapitops.plan.storage.file.SpongePlanFiles.getResourceFromJar(SpongePlanFiles.java:57)
	at com.djrapitops.plan.settings.BukkitConfigSystem.copyDefaults(BukkitConfigSystem.java:78)
	at com.djrapitops.plan.settings.SpongeConfigSystem.copyDefaults(SpongeConfigSystem.java:63)
	at com.djrapitops.plan.settings.ConfigSystem.enable(ConfigSystem.java:75)
	at com.djrapitops.plan.settings.BukkitConfigSystem.enable(BukkitConfigSystem.java:63)
	at com.djrapitops.plan.settings.SpongeConfigSystem.enable(SpongeConfigSystem.java:58)
	at com.djrapitops.plan.PlanSystem.enableSystems(PlanSystem.java:216)
	at com.djrapitops.plan.PlanSystem.enableForCommands(PlanSystem.java:158)
	at com.djrapitops.plan.PlanSystem.enable(PlanSystem.java:210)
	at com.djrapitops.plan.SpongeSystemTest.spongeSystemEnables(SpongeSystemTest.java:57)
```

Reply:
Alright that is fine. I think I made the tests use generic mocked instance to reduce duplication